### PR TITLE
update to a rhel8 base

### PR DESCRIPTION
**Pull request by sbose78**
### Motivation
Downstream is built using rhel8 base, let's do the same upstream too.

### Changes

<< What has been changed in order to achieve "story's" objective? >>

The base image.

### Testing

<< How to test those changes, reference implementation of unit/e2e tests. >>
Once merged, we should test with the latest OperatorSource

### CODE REVIEW

**Comment by sbose78** [comment]
Tested, build has passed in https://quay.io/repository/redhat-developer/app-binding-operator?tab=builds

File: Dockerfile.rhel [hunk 1]
```diff
@@ -32,9 +32,8 @@ RUN make build
 #--------------------------------------------------------------------
 
 # On CI , use
-# FROM registry.svc.ci.openshift.org/ocp/ubi-minimal:7
 
-FROM registry.access.redhat.com/ubi7/ubi-minimal
+FROM registry.access.redhat.com/ubi8/ubi
```
Comment (pedjak) [inline Dockerfile.rhel]: why not `registry.access.redhat.com/ubi8/ubi-minimal`

File: Dockerfile.rhel [hunk 2]
```diff
@@ -32,9 +32,8 @@ RUN make build
 #--------------------------------------------------------------------
 
 # On CI , use
```
Comment (pedjak) [inline Dockerfile.rhel]: I would remove this line as well.

File: Dockerfile.rhel (hunk 1 above)
Comment (sbose78) [reply Dockerfile.rhel]: done https://github.com/redhat-developer/service-binding-operator/pull/316/commits/4c4b2a8ddc0919ec5b401c8b3593e8d91417a15c

File: Dockerfile.rhel (hunk 2 above)
Comment (sbose78) [reply Dockerfile.rhel]: done https://github.com/redhat-developer/service-binding-operator/pull/316/commits/4c4b2a8ddc0919ec5b401c8b3593e8d91417a15c

**Pull request merged**